README.md

```diff
@@ -1,29 +1,27 @@
 > :warning: **This plugin will stop receiving feature requests. I will only spend time for important bug fixes**. Google moved from analytics.js to its new gtag.js library and I've created a new plugin called [vue-gtag](https://github.com/MatteoGabriele/vue-gtag). I suggest you to start using that one if you are about to create a new project.
 
 <p align="center">
-<img src="http://i.imgur.com/whvHAT6.png" width="180" />
-<br>
-<br>
-<a href="https://badge.fury.io/js/vue-analytics">
- <img src="https://badge.fury.io/js/vue-analytics.svg" />
-<a/>
-
- <a href="https://www.npmjs.com/package/vue-analytics">
-  <img src="https://img.shields.io/npm/dm/vue-analytics.svg" />
- <a/>
-
- <a href="https://travis-ci.org/MatteoGabriele/vue-analytics">
-  <img src="https://travis-ci.org/MatteoGabriele/vue-analytics.svg?branch=master" />
- </a>
-
- <img src="https://img.shields.io/badge/size-4kB-brightgreen.svg" />
+  <img width="180" src="http://i.imgur.com/whvHAT6.png">
+  <br>
+  <br>
+  <a href="https://npm.im/vue-analytics">
+    <img src="https://badgen.net/npm/v/vue-analytics">
+  </a>
+  <a href="https://npm.im/vue-analytics">
+    <img src="https://badgen.net/npm/dm/vue-analytics">
+  </a>
+  <a href="https://travis-ci.org/MatteoGabriele/vue-analytics">
+    <img src="https://badgen.net/travis/MatteoGabriele/vue-analytics/master">
+  </a>
+  <a href="https://bundlephobia.com/result?p=vue-analytics">
+    <img src="https://badgen.net/bundlephobia/minzip/vue-analytics">
```

Comment:
as we did in gtag one same mingzip thing?
thanks again for you work

Reply:
I did that already, url resolves to minzipped size badge.